scripts/build_review_app.sh

```diff
@@ -34,29 +35,24 @@ hokusai review_app setup $NAME
 # --tag to name the image.
 # WARNING: This is likely going to take ~10 mins on your MBP.
 # Be patient and grab some baby carrots.
-hokusai registry push --force --skip-latest --verbose --tag $NAME
+hokusai registry push --force --skip-latest --overwrite --verbose --tag $NAME
 
 # Edit the K8S YAML to reference the proper Docker image
-# sed -i '' 's/:staging/:auct-281/g' hokusai/auct-281.yml
-#
-# The weird `-i ''` is needed on MacOS:
-# https://stackoverflow.com/a/44864004
-# WARNING: This might not work on Linux.
-sed -i '' "s/:staging/:$NAME/g" hokusai/$NAME.yml
+sed -i.bak "s/:staging/:$NAME/g" $review_app_file_path && rm $review_app_file_path.bak
```

Comment:
We should wrap any variables in double quotes to prevent globbing and word splitting.

Reply:
`$NAME` is wrapped in double quotes earlier in the script: https://github.com/artsy/force/blob/d3bc6b4d07dcecc30d9869be1d4977d64ce64482/scripts/build_review_app.sh#L17
and `$review_app_file_path` is as well: https://github.com/artsy/force/blob/d3bc6b4d07dcecc30d9869be1d4977d64ce64482/scripts/build_review_app.sh#L17

As such, I think this is fine (aka there's no added risk here)?

Reply:
> As such, I think this is fine (aka there's no added risk here)?

Unfortunately not.

ShellCheck is a great tool for picking up these issues: https://github.com/koalaman/shellcheck

Here's an a simple example:

```bash
$ touch "an example.txt"
$ file="an example.txt"
$ rm 'some example.txt' ^C
$ ls $file
ls: cannot access 'an': No such file or directory
ls: cannot access 'example.txt': No such file or directory
$ ls "$file"
'an example.txt'
```

So even though you've wrapped $1 in double quotes, you should continue to do so for variables passed into other programs.

Reply:
hu interesting! Well TIL. Yeah, I was thinking about it slightly off I guess (values of the vars can't be "escaped once" or anything like that), this makes sense. 👍